subjects/devops/cloud-design/audit/README.md

```diff
@@ -129,7 +120,7 @@ Verify that the student has chosen a cloud platform certification to pursue and
 
 ###### +Is the student prepared and confident in taking the certification exam, and can they apply their knowledge to real-world scenarios?
 
-#### Bonus
+###### +Are alerting mechanisms in place to notify the team of potential issues on time?
 
 ###### +Did the student add any optional bonus?

```

Comment:
this question is included in the ##### Review Cloud Certification Preparation Efforts section

Reply:
In the readme, Alerts are asked as a bonus, so i moved this to the bonus section, should we remove it from being bonus and make this mandatory?

Reply:
no its fine